Flow.Launcher/Themes/BlackAndWhite.xaml

```diff
@@ -18,7 +18,9 @@
         <Setter Property="Background" Value="#000000" />
     </Style>
     <Style x:Key="WindowStyle" TargetType="{x:Type Window}" BasedOn="{StaticResource BaseWindowStyle}" />
-    <Style x:Key="PendingLineStyle" BasedOn="{StaticResource BasePendingLineStyle}" TargetType="{x:Type Line}" />
+    <Style x:Key="PendingLineStyle" BasedOn="{StaticResource BasePendingLineStyle}" TargetType="{x:Type Line}">
```

Comment:
This is the color of the progress bar. The basic color is blue, but there are cases where you can't see it depending on the theme, so I changed a few.

Reply:
if you wanna test, use github plugin and search repo something. github search is too slow. 🐢